### ISSUE TITLE
Accessibility - Web - French - No Visual Indication which Individual Step on Main Steps Page Person is On

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#109]
When a person is tabbing through the main French Steps image, the browser is not showing them the text that used to be displayed to tell them what the Steps icon meant.

**Comment by sanganalakshay** [comment]
fixed

**Comment by jfc3-dol** [comment]
When using IE 11, I can see the individual Steps pop-up message. 

I can not even tab to the Steps in Chrome, and the application will not load in Firefox.

Until we fix the Chrome and Firefox issues, I'm leaving this issue open.

**Comment by sanganalakshay** [comment]
should be fixed

**Comment by jfc3-dol** [comment]
Will test when we get the next code push.

**Comment by jfc3-dol** [comment]
There is now a visual indication for the individual Steps on the main French Steps page for IE 11, Firefox, and Chrome.

I'm going to close this issue.
